recipe/session/sessmodels/models.go

```diff
@@ -178,27 +208,25 @@ type NormalisedErrorHandlers struct {
 }
 
 type TypeSessionContainer struct {
-	RevokeSession            func() error
-	GetSessionData           func() (map[string]interface{}, error)
-	UpdateSessionData        func(newSessionData map[string]interface{}) error
-	GetUserID                func() string
-	GetAccessTokenPayload    func() map[string]interface{}
-	GetHandle                func() string
-	GetAccessToken           func() string
-	UpdateAccessTokenPayload func(newAccessTokenPayload map[string]interface{}) error // Deprecated: use MergeIntoAccessTokenPayload instead
-	GetTimeCreated           func() (uint64, error)
-	GetExpiry                func() (uint64, error)
-
-	RevokeSessionWithContext            func(userContext supertokens.UserContext) error
-	GetSessionDataWithContext           func(userContext supertokens.UserContext) (map[string]interface{}, error)
-	UpdateSessionDataWithContext        func(newSessionData map[string]interface{}, userContext supertokens.UserContext) error
-	GetUserIDWithContext                func(userContext supertokens.UserContext) string
-	GetAccessTokenPayloadWithContext    func(userContext supertokens.UserContext) map[string]interface{}
-	GetHandleWithContext                func(userContext supertokens.UserContext) string
-	GetAccessTokenWithContext           func(userContext supertokens.UserContext) string
-	UpdateAccessTokenPayloadWithContext func(newAccessTokenPayload map[string]interface{}, userContext supertokens.UserContext) error // Deprecated: use MergeIntoAccessTokenPayloadWithContext instead
-	GetTimeCreatedWithContext           func(userContext supertokens.UserContext) (uint64, error)
-	GetExpiryWithContext                func(userContext supertokens.UserContext) (uint64, error)
+	RevokeSession               func() error
```

Comment:
make sure all new functions have withContext versions as well (GetAllSessionTokensDangerously and AttachToRequestResponse had got missed - check again)

Reply:
GetAllSessionTokensDangerously and AttachToRequestResponse dont take user context in the node sdk

https://github.com/supertokens/supertokens-node/pull/531/files#diff-18abb6b9e3d0b3d2a9550565321570f08335a9353218f49910fa41fddbd70c3fR328

https://github.com/supertokens/supertokens-node/pull/531/files#diff-18abb6b9e3d0b3d2a9550565321570f08335a9353218f49910fa41fddbd70c3fR349